react/features/filmstrip/subscriber.web.js

```diff
@@ -74,6 +75,12 @@ StateListenerRegistry.register(
             break;
         case LAYOUTS.VERTICAL_FILMSTRIP_VIEW:
             store.dispatch(setVerticalViewDimensions());
+            if (!shouldDisplayStageFilmstrip(store.getState(), 0)) {
```

Comment:
Isn't this always true?

Reply:
No, but this should be 1 instead of 0. The use case for this is: when we have multiple participants on stage, maybe some pinned, and someone starts screensharing we want to clear the stage participants list.

Reply:
Yes it is. I'll remove it